modules/avcodec/encode.c

```diff
@@ -162,6 +163,10 @@ static int init_encoder(struct videnc_state *st)
 	if (!st->codec)
 		return ENOENT;
 
+	/* TODO: Only set sz_dst when swscale is active.
+	   Is there a way to check if swscale is loaded? */
+	(void)conf_get_vidsz(conf_cur(), "video_size", &sz_dst);
```

Comment:
after baresip has started and parsed the config file, the same config is available
to read from `struct config_video`. Use `conf_config()` to fetch the global config.